tangelo/problem_decomposition/tests/dmet/test_dmet.py

```diff
@@ -55,6 +55,20 @@ def test_incorrect_number_solvers(self):
         with self.assertRaises(RuntimeError):
             DMETProblemDecomposition(opt_dmet)
 
+    def test_not_implemented_solver(self):
+        """Test
+        """
+
+        opt_dmet = {"molecule": mol_H10_321g,
+                    "fragment_atoms": [1]*10,
+                    "fragment_solvers": "banana",
```

Comment:
Just to make sure: this translates as "use the `banana` solver for all 10 fragments" right ?

Reply:
Yes